packages/vitest/src/runtime/collect.ts

```diff
@@ -31,7 +35,13 @@ export async function collectTests(paths: string[], config: ResolvedConfig) {
     clearContext()
     try {
       await runSetupFiles(config)
-      await import(filepath)
+      if (typeof window === 'undefined') {
```

Comment:
maybe it's better to check `config`?

Reply:
config.web can be true in both window === undefined and defined window!

Reply:
Well, config.web can be true in both node and browser! and we only have window defined when we use @vitest/web!